### DISCUSSION TITLE
[Suggestion] Clearer names for 'Editors' and 'Update' tabs

### DISCUSSION THREAD

**Discussion by SixParQuatre** [unitycoder/UnityLauncherPro#179] [Ideas]
I keep on forgetting which one is which :|
I  don't have a strong sense for replacements except maybe, rename 'Editors' to 'Installed Editors' and 'Updates' to 'Unity Versions' or 'Download Unity Versions' ?

**Comment by unitycoder** [comment]
keeping this open, if anyone has good suggestions.

Personally would prefer keeping them roughly same length, so single short word would be perfect.

One other option is to shuffle the order:
Maybe Updates should be where "Settings" is (kind of last item, since its "external content"
and "Settings" should be Aligned to right, to make it separate. (although might be annoying if accidentally press Refresh vs Settings tab at that point)

Also: later "Assets"-tab or something similar would be added, to access your asset store packages.

**Comment by SixParQuatre** [comment]
I  like moving Settings (and even maybe Tools ?) to the right.
For info, the 2 tabs I use daily are Projects and Updates.

**Comment by unitycoder** [comment]
other options:
- Allow reordering those tabs, in settings or drag n drop (could have a look at this, if not too complex)
